packages/math/src/decimal.ts

```
@@ -0,0 +1,35 @@
+import { DEFAULT_MIN_PRECISION, DEFAULT_PRECISION } from './constants';
+import type { ToFixedConfig } from './types';
+
+// export function toFixed(value?: string | number, precision: number = DEFAULT_PRECISION) {
+export function toFixed(value?: string | number, options?: ToFixedConfig) {
+  const { precision = DEFAULT_PRECISION, minPrecision = DEFAULT_MIN_PRECISION } = options || {};
+
+  // todo: minPrecision higher than precision, should override or respect precision?
```

respect the precision... maybe throw a error